mobile/src/modules/transactions/components/send/SendBurstForm.tsx

```diff
@@ -403,6 +405,7 @@ export class SendBurstForm extends React.Component<Props, SendBurstFormState> {
             thumbIconBackgroundColor={Colors.WHITE}
             thumbIconImageSource={actionIcons.chevronRight}
             onSwipeSuccess={this.handleSubmit}
+            shouldResetAfterSuccess={true}
```

Comment:
Fixes an issue where the submit button would be stuck after sending one transaction. (and line 176 below too).